src/Components/Activity/Activity.css

```diff
@@ -3,23 +3,41 @@
     width: 100vw;
     display: flex;
     flex-direction: column;
-    justify-content: center;
+    justify-content: space-evenly;
     align-items: center;
 }
 
-.nav-ber {
+.nav-bar {
     width: 100vw;
-    height: 30%;
+    height: 40px;
+    display: flex;
+    /* flex-direction: column; */
+    flex-wrap: wrap;
+    /* align-content: flex-end; */
+    justify-content: flex-end;
+    margin: 0 0 45px 0;
+}
+
+.favorites-btn {
+    width: 100px;
+    height: 35px;
+    border-radius: 3px;
+    border: transparent;
+    margin: 0 25px;
+    font-size: medium;
 }
+
 .randomizer-container {
     position: relative;
-    /* width: 250px;
-    height: 250px; */ 
     height: 60vh;
     width: 60vw;
     background-color: rgba(21, 24, 35, 0.7);
-    box-shadow: inset 2px 2px 10px rgba(0, 0, 0, 0.3);
-    
+    box-shadow: inset 2px 2px 10px rgb(0 0 0 / 30%);
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    flex-direction: row;
```

Comment:
You actually don't need to call out row here, it defaults to row. But this doesn't effect anything negatively, just something to think about :)